common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -32,8 +32,8 @@
                 this.retrieveFollowed = function() {
                     return DiscussionThreadListView.prototype.retrieveFollowed.apply(self, arguments);
                 };
-                this.chooseCohort = function() {
-                    return DiscussionThreadListView.prototype.chooseCohort.apply(self, arguments);
+                this.chooseGroup = function() {
```

Comment:
Note to reviewers: these front-end changes are for the controls in the Discussion Forums that allow Discussion Admins/Moderators to post to specific groups (formerly just cohorts).